### ISSUE TITLE
Set status payment status on old registrations (from NERD) to "manual"

### ISSUE THREAD

**Issue by odinuge** [webkom/lego#1554]
Only valid registrations with pool on paid events should be marked "manual".

Just like: https://abakus.no/events/1767

![Screenshot from 2019-04-04 15-15-47](https://user-images.githubusercontent.com/1467188/55558519-8ed36500-56ec-11e9-964a-585c6e07e9c4.png)

**Comment by ivarnakken** [comment]
Please enlighten me, but is this still needed now? 😄

**Comment by LudvigHz** [comment]
> Please enlighten me, but is this still needed now? smile @odinuge
> 
> @LudvigHz, you might know?

The reason why this issue is here is that in the event we change some code or decide to check unpaid registrations, all these old things would be included and people could get emails and notifications of unpaid events from ~10 years ago. 

As it is now, it hasn't been an issue. But that's not really a strong argument, so we might still want to look into this. It shouldn't be too hard either. It's just about creating a migration and setting the payment status on all these old registrations.

TLDR; we probably _should_ fix this, but if it's critical? :shrug:
